jobs/search-term-data-validation/ci_job.yaml

```diff
@@ -0,0 +1,15 @@
+build-job-kpi-forecasting:
```

Comment:
This name probably needs to be updated.

Reply:
the name here should match whats on this line https://github.com/mozilla/docker-etl/pull/73/files#diff-99009dbadafe387d6577d23b8496536d46d244c3a91bb56943a44d40a32305faR10

Reply:
Ope, thank you! Good catch.